ui/views_test.py

```diff
@@ -68,3 +72,16 @@ def test_video_detail_unencoded(
     js_settings_json = json.loads(response.context_data['js_settings_json'])
     assert 'videofile' in js_settings_json
     assert js_settings_json['videofile'] == ''
+
+
+def test_video_detail_omniplayer(admin_client, video, videofileHLS, mocker):
+    """Test video detail page when Video.multiangle is True"""
+    mocker.patch('ui.utils.get_cloudfront_signed_url', return_value=videofileHLS.cloudfront_url)
+    video.status = 'Complete'
+    video.multiangle = True
+    video.save()
+    url = reverse('video-detail', kwargs={'pk': video.id})
+    response = admin_client.get(url)
+    assert 'uswitchPlayerURL' in response.context_data
```

Comment:
We should also set a fake value for `settings.USWITCH_URL` so that we know that it is the source of the value here. By default it's an empty string